src/packs/checker.rs

```diff
@@ -339,3 +371,38 @@ fn get_checkers(
         }),
     ]
 }
+
+fn remove_reference_to_dependency(
+    configuration: &Configuration,
+    package_name: &str,
+    dependency_names: &[String],
+) {
+    let existing_pack: &Pack =
+        configuration.pack_set.for_pack(package_name).unwrap();
+    let file_name = existing_pack.yml.clone();
+    let content = fs::read_to_string(&file_name).unwrap();
+
+    let new_content = remove_lines_with_dependency_formatted(
+        content.as_str(),
+        dependency_names,
+    );
+
+    fs::write(&file_name, new_content)
```

Comment:
Could we just call `write_pack_to_disk` directly?

You can take the existing pack and make a new one from it.

Something kind of like this:
```rust
let new_pack = Pack {
  dependencies: new_dependencies,
  ...existing_pack
}

write_pack_to_disk(new_pack)
```

Reply:
That way there's only one way in the codebase we write packs.